stock_available_mrp/migrations/17.0.1.0.0/pre-migration.py

```diff
@@ -0,0 +1,20 @@
+def migrate(cr, version):
+    cr.execute(
+        """SELECT id FROM ir_config_parameter
+        WHERE key = 'stock_available_mrp_based_on'"""
+    )
+    record = cr.fetchone()
+    if record:
+        query = """INSERT INTO ir_model_data (
+            name,
+            model,
+            module,
+            res_id,
+            noupdate)
+            VALUES (
+                'default_stock_available_mrp_based_on',
+                'ir.config_parameter',
+                'stock_available_mrp',
+                '%s',
+                True)"""
+        cr.execute(query, (record[0]))
```

Comment:
I got this error testing this migration script:

```
File "/addons/stock_available_mrp/migrations/17.0.1.0.0/pre-migration.py", line 20, in migrate
    cr.execute(query, (record[0]))
  File "/src/odoo/odoo/sql_db.py", line 327, in execute
    raise ValueError("SQL query parameters should be a tuple, list or dict; got %r" % (params,))
ValueError: SQL query parameters should be a tuple, list or dict; got 39
```

Reply:
Migration SQL query has been solved.